src/Storages/System/StorageSystemServerSettings.cpp

```diff
@@ -18,6 +19,24 @@ namespace CurrentMetrics
 
 namespace DB
 {
+
+static std::vector<std::pair<String, Int8>> getTypeEnumsAndValues()
+{
+    return std::vector<std::pair<String, Int8>>{
+        {"Full",            static_cast<Int8>(RuntimeReloadType::FULL)},
+        {"OnlyIncrease",    static_cast<Int8>(RuntimeReloadType::ONLY_INCREASE)},
+        {"OnlyDecrease",    static_cast<Int8>(RuntimeReloadType::ONLY_DECREASE)},
+        {"No",              static_cast<Int8>(RuntimeReloadType::NO)},
+    };
+}
+
+struct UpdatedData {
+    std::string value;
+    RuntimeReloadType type;
+};
+
+
+
```

Comment:
Please add docs for `runtime_reload` to docs/en/operations/system-tables/server_settings.md.

For some reason, existing column `is_hot_reloadable` is undocumented (in the same markdown document). While at it, would you please add some docs for it as well?